kinto/tests/test_views_flush.py

```diff
@@ -50,7 +47,6 @@ def test_returns_404_if_not_enabled_in_configuration(self):
         app = self._get_test_app(settings=extra)
         app.post('/__flush__', headers=self.headers, status=404)
 
-    @authorize(authz_class='kinto.tests.support.AllowAuthorizationPolicy')
```

Comment:
This should be true with the current kinto authorization, since __flush__ doesn't need any permission (NO_PERMISSION_REQUIRED)